TYPO3.Flow/Classes/TYPO3/Flow/I18n/Translator.php

```diff
@@ -166,6 +166,12 @@ public function translateByOriginalLabel($originalLabel, array $arguments = [],
      */
     public function translateById($labelId, array $arguments = [], $quantity = null, Locale $locale = null, $sourceName = 'Main', $packageKey = 'TYPO3.Flow')
     {
+        $combinedIdentifiers = explode(':', $labelId);
```

Comment:
It was supposed to support stuff like <f:translate id="Package:Source:TranslationUnitId" />
Thinking about XLIFF 2.0, where files have identifiers themselves and can be resolved without a package name or source whatsoever, this seems no longer necessary. But that leads to a different topic